webapp/app/templates/eligibility/form_full_width.html

```diff
@@ -10,4 +10,6 @@
     {% for field in form %}
         {{ macros.render_field(field, 12, position_details_after=True) }}
     {% endfor %}
+    {% block additional_form_content %}
+    {% endblock %}
```

Comment:
You do not need this. Just use `block form_content` with super in form_marital_status_input.html